CarpetXRegrid/interface.ccl

```diff
@@ -0,0 +1,9 @@
+# Interface definition for thorn Regrid
+implements: CarpetXRegrid
```

Comment:
We usually use all caps keywords, i.e. `IMPLEMENTS`.

They keyword `inherits` is not necessary since nothing is inherited.

Reply:
will do